mypy/experimental.py

```diff
@@ -0,0 +1 @@
+STRICT_OPTIONAL = False
```

Comment:
I don't think an adjective make a good module name. Can you make it a noun, e.g. 'experiments'?